training/model_evaluation/components/pipeline_component/spec.yaml

```diff
@@ -83,7 +85,7 @@ outputs:
 jobs:
   model_prediction:
     type: command
-    component: azureml://registries/azureml/components/model_prediction/versions/0.0.5
+    component: azureml://registries/azureml-preview/components/model_prediction/versions/0.0.6
```

Comment:
zureml://registries/azureml/components/model_prediction/versions/0.0.6 *
preview should be removed

Reply:
removed